tests/FlatBuffers.Test/Resources/monsterdata_test.json

```diff
@@ -0,0 +1 @@
+../../monsterdata_test.json
```

Comment:
what are these? just links? Why does whoever uses them not refer to the original?

Reply:
Yeah, they were originally copied to a new temp directory and deleted. So I just converted to links to avoid those steps. But it  probably is best to just reference the original files directly.